packages/account/src/Sections/Security/LoginHistory/login-history-table-row.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Table, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+
+export type TLoginHistoryItems = {
+    id: number;
+    date: string;
+    action: string;
+    browser: string;
+    ip: string;
+    status: string;
+};
```

Comment:
This type is used in  `packages/account/src/Sections/Security/LoginHistory/login-history-list-row.tsx`
Please move it to packages/account/src/Types/common-prop.type.ts